src/pages/groupComparison/GroupComparisonMutationsTabPlot.tsx

```diff
@@ -107,6 +111,11 @@ export default class GroupComparisonMutationsTabPlot extends React.Component<
         }
     }
 
+    @action.bound
+    protected onClickSettingMenu(visible: boolean) {
```

Comment:
i don't understand why the plot needs to know about settings menu?  it seems like a concern of the parent tab component and indeed that's where the settings button is.

Reply:
Actually we have multiple settings buttons. There is one hidden in the filter panel, and the same button appears in the tooltip as well. So, 3 buttons for the same functionality :)

![image](https://user-images.githubusercontent.com/3604198/211110631-b7e5cba9-3872-44be-a90a-a7c0503e345b.png)

The component name `plot` is a bit confusing here. This component is not just the lollipop plot, it also contains the filter panel, 3D visualzier, etc. It's basically still a MutationMapper component with the table disabled.